build/resin-browser.js

```diff
@@ -1599,7 +1599,7 @@ getDeviceModel = function(deps, opts) {
         return request.send({
```

Comment:
Need to check again how GH detects the generated files and add some line that will please it.